docs/configuration/configuration.md

```diff
@@ -1501,6 +1505,69 @@ See [the Prometheus eureka-sd configuration file](/documentation/examples/promet
 for a practical example on how to set up your Eureka app and your Prometheus
 configuration.
 
+### `<scaleway_sd_config>`
+
+The following meta labels are available on targets during [relabeling](#relabel_config):
+
+#### Instance role
+
+* `__meta_scaleway_instance_id`: the id of the instance
+* `__meta_scaleway_instance_ip_is_public`: whether the ip of the target is publicly routed
+* `__meta_scaleway_instance_ip_version`: version of the ip `IPv4`, `IPv6`
+* `__meta_scaleway_instance_image_name`: name of the server image
+* `__meta_scaleway_instance_name`: name of the server
+* `__meta_scaleway_instance_status`: status of the server
+* `__meta_scaleway_instance_tags`: tag list of the server
+* `__meta_scaleway_instance_type`: commercial type of the server
+* `__meta_scaleway_instance_zone`: zone of the instance (`fr-par-1`, `fr-par-2`, `nl-ams-1`, `pl-waw-1`)
```

Comment:
I would give an exemple but not list them all, to avoid going out of date.

Reply:
Is there automated reference documentation that would register all possible meta? If there is one I can add a link to it to have the exhaustive list. Otherwise, I think it is best to have all the labels and enforce to keep them up to date.